server.js

```diff
@@ -64,6 +74,36 @@ if (localConfigExists) {
   }
 }
 
+const profileUrl = `http://${host}:${port}`;
+if (argv.profile) {
+  // Not modifying serverConfig.static because that can roughly triple
+  // webpack cached build time.
+  const prevSetupMiddlewares = serverConfig.setupMiddlewares;
+  serverConfig.setupMiddlewares = (middlewares, devServer) => {
+    if (prevSetupMiddlewares) {
+      middlewares = prevSetupMiddlewares(middlewares, devServer);
+    }
+    devServer.app.use(
+      '/profiles/',
+      express.static(path.resolve(path.dirname(argv.profile)))
+    );
```

Comment:
This does expose every file under `path.dirname(argv.profile)`. It doesn't seem too serious for its intended use case but will probably need mitigation soon just in case someone changes the host away from localhost.

Reply:
Mitigation is done by filtering (cf97a59a425d3d03f05024d0d9b72fe7bf6f33c9)